### PR TITLE
No Conflicts

### PR DESCRIPTION
**Pull request by rokob**
Create a `noconflict` bundle which can be used to bundle rollbar inside a component intended to be embedded on someone's site.

### CODE REVIEW

File: src/browser/globalSetup.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-function captureUncaughtExceptions(window, handler) {
```
Comment (rokob) [inline src/browser/globalSetup.js]: This file has real code changes

File: src/browser/shim.js [hunk 2]
```diff
@@ -46,12 +46,12 @@ function setupShim(window, options) {
   return _wrapInternalErr(function() {
```
Comment (rokob) [inline src/browser/shim.js]: This file has real changes

File: README.md [hunk 3]
```diff
@@ -712,6 +712,45 @@ window.Rollbar.configure({
 ```
 
 
+## Using inside an embedded component
```
Comment (rokob) [inline README.md]: This is new

**Comment by rokob** [comment]
Most of this diff is generated code, I marked with comments the stuff that is actually causing the changes.

File: src/browser/globalSetup.js [hunk 4]
```diff
@@ -34,10 +34,10 @@ function _rollbarWindowOnError(window, r, old, args) {
   }
 }
 
-function captureUnhandledRejections(window, handler) {
+function captureUnhandledRejections(window, handler, shim) {
   if (!window) { return; }
 
-  if (typeof window._rollbarURH === 'function') {
+  if (typeof window._rollbarURH === 'function' && window._rollbaarURH.belongsToShim) {
```
Comment (coryvirok) [inline src/browser/globalSetup.js]: Typo

**Pull request merged**